src/vs/base/browser/ui/dropdown/dropdownActionViewItem.ts

```diff
@@ -57,11 +58,17 @@ export class DropdownMenuActionViewItem extends BaseActionViewItem {
 
 	render(container: HTMLElement): void {
 		const labelRenderer: ILabelRenderer = (el: HTMLElement): IDisposable | null => {
-			this.element = append(el, $('a.action-label.codicon')); // todo@aeschli: remove codicon, should come through `this.options.clazz`
-			if (this.options.clazz) {
-				addClasses(this.element, this.options.clazz);
+			this.element = append(el, $('a.action-label'));
+
+			const classNames = this.options.classNames ? asArray(this.options.classNames) : [];
+
+			// todo@aeschli: remove codicon, should come through `this.options.classNames`
+			if (!classNames.find(c => c === 'icon')) {
+				classNames.push('codicon');
 			}
 
+			addClasses(this.element, ...classNames);
```

Comment:
fyi - es6 deprecated, use `this.element.classList...`

Reply:
Actually didn't know that!